app/services/terms_of_service.rb

```diff
@@ -0,0 +1,11 @@
+class TermsOfService
+  def self.tos_accepted?(customer, distributor = nil)
+    return false unless accepted_at = customer&.terms_and_conditions_accepted_at
+
+    if distributor
+      accepted_at > distributor.terms_and_conditions_updated_at
```

Comment:
Codecov says there's no coverage on this line...

Reply:
Should have it now 👍